code/modules/religion/religion_sects.dm

```diff
@@ -88,132 +98,292 @@
-	if(!ishuman(L))
+/datum/religion_sect/proc/sect_bless(mob/living/chap, mob/living/user)
+	if(!ishuman(chap))
 		return FALSE
-	var/mob/living/carbon/human/H = L
-	for(var/X in H.bodyparts)
-		var/obj/item/bodypart/BP = X
-		if(BP.status == BODYPART_ROBOTIC)
+	var/mob/living/carbon/human/blessed = chap
+	for(var/X in blessed.bodyparts)
+		var/obj/item/bodypart/bodypart = X
+		if(bodypart.status == BODYPART_ROBOTIC)
 			to_chat(user, "<span class='warning'>[GLOB.deity] refuses to heal this metallic taint!</span>")
 			return TRUE
 
 	var/heal_amt = 10
-	var/list/hurt_limbs = H.get_damaged_bodyparts(1, 1, null, BODYPART_ORGANIC)
+	var/list/hurt_limbs = blessed.get_damaged_bodyparts(1, 1, null, BODYPART_ORGANIC)
 
 	if(hurt_limbs.len)
 		for(var/X in hurt_limbs)
 			var/obj/item/bodypart/affecting = X
 			if(affecting.heal_damage(heal_amt, heal_amt, null, BODYPART_ORGANIC))
-				H.update_damage_overlays()
-		H.visible_message("<span class='notice'>[user] heals [H] with the power of [GLOB.deity]!</span>")
-		to_chat(H, "<span class='boldnotice'>May the power of [GLOB.deity] compel you to be healed!</span>")
+				blessed.update_damage_overlays()
+		blessed.visible_message("<span class='notice'>[user] heals [blessed] with the power of [GLOB.deity]!</span>")
+		to_chat(blessed, "<span class='boldnotice'>May the power of [GLOB.deity] compel you to be healed!</span>")
 		playsound(user, "punch", 25, TRUE, -1)
-		SEND_SIGNAL(H, COMSIG_ADD_MOOD_EVENT, "blessing", /datum/mood_event/blessing)
+		SEND_SIGNAL(blessed, COMSIG_ADD_MOOD_EVENT, "blessing", /datum/mood_event/blessing)
 	return TRUE
 
 /datum/religion_sect/puritanism
 	name = "Puritanism (Default)"
 	desc = "Nothing special."
-	convert_opener = "Your run-of-the-mill sect, there are no benefits or boons associated. Praise normalcy!"
+	convert_opener = "\"Praise normalcy!\"<br>\
+	Your run-of-the-mill sect, there are no benefits or boons associated."
+
+/**** Mechanical God ****/
 
-/datum/religion_sect/technophile
-	name = "Technophile"
+/datum/religion_sect/mechanical
+	name = "Mechanical God"
 	desc = "A sect oriented around technology."
-	convert_opener = "May you find peace in a metal shell, acolyte.<br>Bibles now recharge cyborgs and heal robotic limbs if targeted, but they do not heal organic limbs. You can now sacrifice cells, with favor depending on their charge."
+	convert_opener = "\"May you find peace in a metal shell.\"<br>\
+	Bibles now recharge cyborgs and heal robotic limbs if targeted, but they do not heal organic limbs. \
+	You can now sacrifice cells, with favor depending on their charge."
 	alignment = ALIGNMENT_NEUT
 	desired_items = list(/obj/item/stock_parts/cell)
 	rites_list = list(/datum/religion_rites/synthconversion)
 	altar_icon_state = "convertaltar-blue"
 
-/datum/religion_sect/technophile/sect_bless(mob/living/L, mob/living/user)
-	if(iscyborg(L))
-		var/mob/living/silicon/robot/R = L
+/datum/religion_sect/mechanical/sect_bless(mob/living/chap, mob/living/user)
```

Comment:
chap isnt accurate here, L is your poor soul. user would be the chaplain.

Reply:
this is why we don't use single letter vars 😢